torch/fx/node.py

```diff
@@ -914,23 +913,4 @@ def map_aggregate(a: ArgumentT, fn: Callable[[Argument], Argument]) -> ArgumentT
     arg may be a list, tuple, slice, or dict with string keys: the return value will
     have the same type and structure.
     """
-    result: Argument
-
-    if isinstance(a, tuple):
-        it = (map_aggregate(elem, fn) for elem in a)
-        # Support NamedTuple (if it has `_fields`) by repacking into original type.
-        result = type(a)(*it) if hasattr(a, "_fields") else tuple(it)
-    elif isinstance(a, list):
-        result = immutable_list([map_aggregate(elem, fn) for elem in a])
-    elif isinstance(a, dict):
-        result = immutable_dict([(k, map_aggregate(v, fn)) for k, v in a.items()])
```

Comment:
One blocker to use the pytree `tree_map` is that pytree uses `type(a) is list` / `type(a) is dict` rather than `isinstance(a, list)` / `isinstance(a, dict)`. How do we want to support user subclasses, such as `collections.UserList` / `collections.UserDict`? Or we can just support `list` (`immutable_list`), `dict` (`immutable_dict`), `defaultdict`, `OrderedDict` only.